shared/ui/Stream/PullRequests/Bitbucket/MergeScreen.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from "react";
+import { api } from "../../../store/providerPullRequests/thunks";
+import { useAppDispatch } from "@codestream/webview/utilities/hooks";
+import { FetchThirdPartyPullRequestPullRequest } from "@codestream/protocols/agent";
+
+import { Dialog } from "../../../src/components/Dialog";
+import Button from "../../Button";
+import { InlineMenu } from "@codestream/webview/src/components/controls/InlineMenu";
+import { Modal } from "../../Modal";
+import { WarningBox } from "../../WarningBox";
+
+interface Props {
+	pr: FetchThirdPartyPullRequestPullRequest;
+	onClose: Function;
```

Comment:
Function is the `any` of functions - can type it pretty easily
```suggestion
	onClose: () => void;
```